src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -68,7 +62,7 @@ const General = ({
     },
   ];
 
-  const [selectedProfile, setSelectedProfile] = useState(data.profile);
```

Comment:
why not just leave the initial state as data.profile ? incase it does resolve fast enough